File: java/chainofresponsibility/Response.java
package chainofresponsibility;

public class Response {
    private String reason;
    private boolean isSucceeded;

    public Response(String reason, boolean isSucceeded) {
        this.reason = reason;
        this.isSucceeded = isSucceeded;
    }

    public String getReason() {
        return reason;
    }

    public boolean isSucceeded() {
        return isSucceeded;
    }
}

File: PHP/Patterns/src/Behavioral/ChainOfResponsibility/Response.php
<?php

namespace src\Behavioral\ChainOfResponsibility;

class Response
{
    private $message;
    private $status;

    /**
     * @param $message
     * @param $status
     */
    public function __construct($message, $status)
    {
        $this->message = $message;
        $this->status = $status;
    }

    /**
     * @return mixed
     */
    public function getMessage()
    {
        return $this->message;
    }

    /**
     * @param mixed $message
     */
    public function setMessage($message): void
    {
        $this->message = $message;
    }

    /**
     * @return mixed
     */
    public function getStatus()
    {
        return $this->status;
    }

    /**
     * @param mixed $status
     */
    public function setStatus($status): void
    {
        $this->status = $status;
    }
}